setup.cfg

```diff
@@ -1,7 +1,7 @@
 [tool:pytest]
 addopts = -v --junit-xml=test-report.xml
           --doctest-modules
-          --cov=riotctrl --cov-branch
+          --cov=riotctrl --cov-branch --cov-append
```

Comment:
Could it rather be called as an additional posargs from the github action ? Because otherwise I fear side effect when run locally if the developer forget to remove the coverage.xml file.

Reply:
Yeah, I was thinking the same but did not dare to touch your nicely cleaned up tox calls without permission :D

Reply:
Done in 89d86af